code/modules/antagonists/nations/factions/security.dm

```diff
@@ -0,0 +1,19 @@
+/datum/team/nationsec
+	name = "Securitystan"
```

Comment:
Please ask people to help you come up with better names

Reply:
Securistan or Sectopia at least